Trend Micro/cloud-one/tests/SystemEventLog.json

```diff
@@ -0,0 +1,46 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cloud One",
+        "dialect_uuid": "b2d961ae-0f7e-400b-879a-f97be24cc02d"
+      }
+    },
+    "message": "CEF:0|Trend Micro|Workload Security Manager|<Workload Security version>|600|User Signed In|3|src=10.52.116.160 suser=admin target=admin msg=User signed in from 2001:db8::5"
+  },
+  "expected": {
+    "message": "CEF:0|Trend Micro|Workload Security Manager|<Workload Security version>|600|User Signed In|3|src=10.52.116.160 suser=admin target=admin msg=User signed in from 2001:db8::5",
```

Comment:
Not an intrusion detection but a sign-in.
The category of the event should be ['authentication'] and the type ['start']

Reply:
Fixed but we will need more log to be more precise